src/templates/fr/2019/base.html

```diff
@@ -102,7 +102,7 @@
 %}
 
 {% set localizedContributors = {
-  "kleinab": "<i lang=\"en\"><i lang=\"en\">Abigail Klein is a Google software engineer. She worked on Google Docs, Sheets, and Slides web accessibility where she added <a href=\"https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/\">automatic captions to Google Slides</a>, as well as improving screen reader, braille, screen magnifier, and high contrast support. She currently works on Google Chrome and ChromeOS accessibility. She has a bachelor's and master's degree in computer science from MIT, where she co-founded an assistive technology hackathon and was a lab assistant and guest lecturer of the assistive technology class.</i>",
+  "kleinab": "Abigail Klein est <i lang="en">software engineer</i> chez Google. Elle a travaillé sur l’accessibilité web de Google Docs, Sheets et Slides où elle a ajouté les <a href=\"https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/\">légendes automatiques de Google Slides</a>, ainsi que l’amélioration du lecteur d’écran, du braille, de la loupe d’écran et de la prise en charge du contraste élevé. Elle travaille actuellement sur l’accessibilité de Google Chrome et ChromeOS. Elle détient un baccalauréat et une maîtrise en informatique du MIT, où elle a cofondé un hackathon de technologie d’assistance et a été assistante de laboratoire et conférencière invitée de la classe de technologie d’assistance.",
```

Comment:
Need a small tweak to this to escape the quotes:

```suggestion
  "kleinab": "Abigail Klein est <i lang=\"en\">software engineer</i> chez Google. Elle a travaillé sur l’accessibilité web de Google Docs, Sheets et Slides où elle a ajouté les <a href=\"https://www.blog.google/outreach-initiatives/accessibility/whats-you-say-present-captions-google-slides/\">légendes automatiques de Google Slides</a>, ainsi que l’amélioration du lecteur d’écran, du braille, de la loupe d’écran et de la prise en charge du contraste élevé. Elle travaille actuellement sur l’accessibilité de Google Chrome et ChromeOS. Elle détient un baccalauréat et une maîtrise en informatique du MIT, où elle a cofondé un hackathon de technologie d’assistance et a été assistante de laboratoire et conférencière invitée de la classe de technologie d’assistance.",
```

Reply:
Well, @borisschapira fixed it in his PR :D